flask_resources/parsers/parsers.py

```diff
@@ -72,4 +75,9 @@ def parse(self):
         if self.argmap is None:
             return {}
         flaskparser = ResourceFlaskParser()
-        return flaskparser.parse(self.argmap, request, locations=("querystring",))
+        try:
+            return flaskparser.parse(self.argmap, request, locations=("querystring",))
+        except BadRequest:
+            raise BadRequestRESTError(
+                    description="Cannot parse url's querystring"
+            )
```

Comment:
Perhaps this needs to be checked a bit more. If we're change it to return 400 somewhere (which I think Guillaume said we did), then likely, it's better not to change it there, and instead of ``BadRequest`` catch something else.

Reply:
Yes we are changing the default webargs exception error code here: https://github.com/inveniosoftware/flask-resources/blob/master/flask_resources/parsers/parsers.py#L19-L22